R/map.R

```diff
@@ -41,22 +41,31 @@
 #' @seealso [map_if()] for applying a function to only those elements
 #'   of `.x` that meet a specified condition.
 #' @examples
+#' # Compute normal distributions from an atomic vector
 #' 1:10 %>%
-#'   map(rnorm, n = 10) %>%
-#'   map_dbl(mean)
+#'   map(rnorm, n = 10)
 #'
-#' # Or use an anonymous function
+#' # You can also use an anonymous function
 #' 1:10 %>%
 #'   map(function(x) rnorm(10, x))
 #'
 #' # Or a formula
 #' 1:10 %>%
 #'   map(~ rnorm(10, .x))
 #'
+#' # Reduce output to a vector instead of a list by computing the mean of the distributions
```

Comment:
"Reduce" might be confusing here because of `purrr::reduce()`. Perhaps "Simplify"?

Also should mention that `map_dbl()` is the function that creates an atomic vector of doubles whereas `map()` always creates a list.

Reply:
would this work?
```r
#' # Simplify output to a vector instead of a list by computing the mean of the distributions
#' 1:10 %>%
#'   map(rnorm, n = 10) %>%  # outputs a list
#'   map_dbl(mean)           # outputs an atomic vector
```